.travis.yml

```
@@ -1,10 +1,16 @@
 sudo: false
 language: python
+env:
+    - PYCURL_SSL_LIBRARY=gnutls
 python:
     - "3.6"
+before_install:
+    - sudo apt-get install -y libgnutls-dev
 install:
     - pip install -U pip
     - pip install -r requirements.txt codecov flake8 sphinx tox-travis manage pre-commit
+    - pip uninstall -y pycurl
+    - pip install --compile --no-cache-dir pycurl
```

```suggestion
    - pip install -U pip 
    - pip uninstall -y pycurl
    - pip install --compile --no-cache-dir pycurl
    - pip install -r requirements.txt codecov flake8 sphinx tox-travis manage pre-commit 
```

I think there should not be any harm to add these steps (unless anything more related to pycurl compile/recompile) ..
However could you please set the in above order and try once more to see if all works fine ?

---

It makes no difference for travis.
It also makes no difference for me. It can be this way, still works.

---

the idea was to see if compile works perfectly in different order too  before install the `pip install -r requirements.txt codecov flake8 sphinx tox-travis manage pre-commit `

as travis is looks satisfied hence LGTM :+1: